integration-test/test-pages/broker-protection/actions/click-weighted.json

```diff
@@ -0,0 +1,42 @@
+{
+  "state": {
+    "action": {
+      "actionType": "click",
+      "elements": [
+        {
+          "type": "button",
+          "selector": ".opt-out",
+          "parent": {
+            "profileMatch": {
+              "selector": ".record",
+              "profile": {
+                "name": {
+                  "selector": ".name"
+                },
+                "age": {
+                  "selector": ".age"
+                },
+                "addressCityState": {
+                  "selector": ".address"
+                }
+              }
+            }
+          }
+        }
```

Comment:
and this example shows how to match an outer element first, and then perform a sub-query inside.